config.go

```diff
@@ -524,7 +524,7 @@ func (config *TGFConfig) findRemoteConfigFiles(location, files string) []string
 		log.Debugln("Reading configuration from", fullConfigPath)
 		source := must(getter.Detect(fullConfigPath, must(os.Getwd()).(string), getter.Detectors)).(string)
 
-		err := getter.Get(destConfigPath, source)
+		err := getter.GetFile(destConfigPath, source)
```

Comment:
From the documentation, GetAny seems to be more appropriate.

Reply:
Considering that `Get` is supposed to only support directories, we would be switching from "dir only" to "file only" instead of "dir or file". I agree with this :+1:

Reply:
I thought so too, but when I tried it copied in a way that caused other runtime errors.